### DISCUSSION TITLE
Running Unprivileged in Azure - unable to port map

### DISCUSSION THREAD

**Discussion by kamikaze138** [nginx/docker-nginx-unprivileged#172] [Q&A] [answered]
Unlike Docker/Podman, [Azure Container Instances doesn't permit port mapping](https://social.msdn.microsoft.com/Forums/windowsdesktop/en-US/ff3550c3-3ea7-4334-a4c9-eb78b6c1e9d5/how-to-do-port-mapping-in-azure-how-to-run-docker-equivalent-command-on-azure-while-creating-a?forum=AzureContainerServices#:~:text=Port%20mapping%20is%20not%20yet%20supported%2C%20current%20work,through%20which%20the%20port%20mapping%20can%20be%20done.).  With the inability to set 

```
- ports:
    8443:443
    8080:80
```
I receive this message:

```
2023/10/31 14:39:18 [emerg] 54#54: bind() to 0.0.0.0:443 failed (13: Permission denied)
nginx: [emerg] bind() to 0.0.0.0:443 failed (13: Permission denied)
```

The Dockerfile is using : _nginxinc/nginx-unprivileged:stable-alpine_

I've attempted various changes to EXPOSE - attempting to run it as User root , as well as the desired end-state non-privileged account.   
 I have also attempted to edit the internal _/etc/sysctl.conf_ [file upon build](https://github.com/jlesage/docker-nginx-proxy-manager/issues/117#issuecomment-862764309) to include `net.ipv4.ip_unprivileged_port_start=0` However, upon research, ports 1024 and below are reserved.  I can live with using port 8080. However, if I am to deliver a site via SSL, how can that be achieved without the port mapping ?

**Comment by alessfg** [comment, marked as answer]
From what I can find, you might be able to try something like what's described here https://stackoverflow.com/a/75031898, but I am also seeing reports of ACI not allowing HTTPS/SSL connections, and needing to create an additional sort of entrypoint/sidecar https://stackoverflow.com/a/60965775. I'm sorry I can't offer any further assistance!
